packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -246,17 +246,16 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     const i = bisect(lineChartData![0].data, xOffset);
     const d0 = lineChartData![0].data[i - 1] as ILineChartDataPoint;
     const d1 = lineChartData![0].data[i] as ILineChartDataPoint;
-    let axisType: XAxisTypes | null = null;
     let pointToHighlight: string | Date | number | null = null;
     let index: null | number = null;
+    const axisType = getTypeOfAxis(lineChartData![0].data[0].x, true) as XAxisTypes;
```

Comment:
add a check that lineChartData![0].data.length > 0. If == 0, let the axisType be null.

Reply:
Done